assets.md

```diff
@@ -9,6 +9,7 @@ This document outlines the assets used by the OpenIntegrationEngine (OIE) projec
 | GitHub Organization      | [OpenIntegrationEngine](https://github.com/OpenIntegrationEngine) | Kaur Palang                | Yes                                                 |
 | Code Signing Certificate | *None listed*                                                    | BrightCodeCompany OÜ       | No<sup>*</sup>                                      |
 | Docker Hub account       | [OpenIntegrationEngine](https://hub.docker.com/u/openintegrationengine)   | BrightCodeCompany OÜ    | Yes                                                 |
+| LinkedIn                 | [Open Integration Engine](https://www.linkedin.com/company/open-integration-engine/) | Chris Gibson | Yes |                      
```

Comment:
I think the link should not have the trailing slash. While signed out of LinkedIn, without the slash it showed me the company page. With the slash it sends me to a login screen.